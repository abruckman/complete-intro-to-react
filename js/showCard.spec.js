import React from 'react'
import ShowCard from './showCard'
import {shallow} from 'enzyme'
import {shallowToJson} from 'enzyme-to-json'
import preload from '../public/data.json'


test('Search snapshot test', ()=> {
  const show = preload.shows[0]
  const component = shallow(<ShowCard key={show.imdbID} {...show} />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
