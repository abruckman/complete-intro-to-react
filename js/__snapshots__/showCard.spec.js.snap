// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot test 1`] = `
<Link
  activeClassName=""
  activeOnlyWhenExact={false}
  activeStyle={Object {}}
  className=""
  isActive={[Function]}
  replace={false}
  style={Object {}}
  to="/details/tt1856010"
>
  <div
    className="show-card"
  >
    <img
      src="/public/img/posters/hoc.jpg"
    />
    <h3>
      House of Cards
    </h3>
    <h4>
      (
      2013â€“
      )
    </h4>
    <p>
      A Congressman works with his equally conniving wife to exact revenge on the people who betrayed him.
    </p>
  </div>
</Link>
`;
